src/app/Fake.DotNet.Cli/DotNet.fs

```diff
@@ -4,20 +4,25 @@ namespace Fake.DotNet
 /// .NET Core + CLI tools helpers
 /// </summary>
 [<RequireQualifiedAccess>]
+#if FAKE_INTERNAL_DOTNET_CORE_CLI
+module InternalDotNet =
```

Comment:
This was because when I tried making the module internal inside Fake.Runtime I got 
![image](https://github.com/user-attachments/assets/63c2be3e-0a8c-4226-870b-695a95e3c049)
so the name was an attempt at avoiding any clashes